### ISSUE TITLE
3.5.5a "SpellChainEffects.dbc" field Combo 

### ISSUE THREAD

**Issue by ladiszlai** [WowDevTools/WDBXEditor#36]
This awesome program is the only one who can read this dbc with almost every field expect this 'combo'. 
Can you guys please fix it ? 

![bug](https://user-images.githubusercontent.com/6253595/35769740-e160d2ac-090f-11e8-8aa0-fc684bc1896a.jpg)

**Comment by barncastle** [comment]
I've spoken to some people more knowledgeable than me and it appears that Blizzard have done something strange here, this is the correct value however they're storing data as a string that shouldn't be strings however we have no idea what just yet. 

I'll leave this open for the time being.

**Comment by barncastle** [comment]
I've spent some time looking at this and digging through the client and it appears that Blizzard are in fact storing a byte array as a string - although I'm not 100% sure what this array actually affects.

I'm on the fence about actually adding a custom editor for this as it is a lot of work for a single field, only found in WotLK and Cata, and which is pretty much undocumented indicating no-one edits this... Is it something that is vital for what you're trying to achieve?

**Comment by ladiszlai** [comment]
Thank you for your research !
This combo field is for multiple chain effect. In 6.0.1 it changed to "SpellChainEffectID" so here we can just put the next chain effect ID. In wotlk I don't know how the combo field refers to the next chain effect ID...

**Comment by BarbzYHOOL** [comment]
What is it used for ladiszlai ? what will you use it for ?

I confirm I can't open it with mydbceditor at all

**Comment by ladiszlai** [comment]
I will use it to create new spell chain effect.. what else ?

**Comment by BarbzYHOOL** [comment]
So you're the first guy in all these years to try to do this?? it kind of surprises me, or else other people found another way

**Comment by ladiszlai** [comment]
I think no one wanted to create chain effects.

**Comment by barncastle** [comment]
As far as I'm concerned, WDBX is reading and saving the field correctly anything more than that is just "nice-to-have". Additionally, as we don't currently know how this is used by the client or what valid values are I can't build a tool around editing it.

So for the meantime I'm going to close this. @ladiszlai if you figure out how this works, reopen this ticket and I'll look into implementing a custom tool/editor. 

Hopefully that all sounds fair.

**Comment by barncastle** [comment]
@ladiszlai It has been a while but I've finally figured this out! I've created a quick'n'dirty [python script](https://gist.github.com/barncastle/7ba656b732efdf9f13866870406c83ed) for encoding/decoding these values, I've also documented the important points on the [wiki](https://wowdev.wiki/DB/SpellChainEffects#m_Combo).

Hopefully this helps, one day I'll add this to WDBX as well - but unfortunately not today.
